src/rockstor/storageadmin/views/rockon_id.py

```diff
@@ -145,13 +145,11 @@ def post(self, request, rid, command):
                         cco.val = cc_map[c]
                         cco.save()
                     for e in env_map.keys():
-                        if (not DContainerEnv.objects.filter(
+                        if (DContainerEnv.objects.filter(
```

Comment:
Thank you, @MFlyer, I didn't see it like a judgement. I just wanted to show that removing an error check is not a 'hack' or something but a real solution :) I will try to "master this issue" and moreover I've already found another bug in the same function so I'll better change original pr a little.